src/main/java/cceclipseplugin/editor/DocumentManager.java

```diff
@@ -99,6 +107,54 @@ public void closedDocument(String absolutePath) {
 			setCurrFile(null);
 		}
 		this.openEditors.remove(absolutePath);
+		this.sendFilePullRequest(absolutePath);
+	}
+	
+	private void sendFilePullRequest(String absolutePath) {
+		PluginManager pm = PluginManager.getInstance();
+		
+		String workspaceRoot = ResourcesPlugin.getWorkspace().getRoot().getLocation().toString();
+		java.io.File rootFile = new java.io.File(workspaceRoot);
+		java.io.File absoluteFile = new java.io.File(absolutePath);
+		String workspaceRelativePathString = "/" + rootFile.toURI().relativize(absoluteFile.toURI()).toString();
+		IPath relativePath = new Path(workspaceRelativePathString);
+		
+		FileMetadata file = pm.getMetadataManager().getFileMetadata(workspaceRelativePathString);
+		if (file == null) {
+			return; // not a tracked file
```

Comment:
What do you think about adding a sysout here that will eventually get converted to a logged statement? I started doing that for these null metadata checks on my most recent branch and it seems like good practice.